cocos/physics/ammo/ammo-const.ts

```diff
@@ -12,6 +12,7 @@ export const CollisionEventObject = {
     selfCollider: null as unknown as ColliderComponent,
     otherCollider: null as unknown as ColliderComponent,
     contacts: [] as IContactEquation[],
+    isA: false,
```

Comment:
The naming is strange,  how about `first` & `second`? or `bodyA` & `bodyB` ?

Reply:
bullet and cannon used `A B` to identify the body, so I think `A` is not bad.